ansible/my-ansible-playbook.yml

```
@@ -0,0 +1,67 @@
+- name: Exercise 4 Playbook - Setup Web Servers
+  hosts: managed_nodes
+  remote_user: ec2-user
+
+  vars_prompt:
+  - name: trello_api_key
+    prompt: What is your TRELLO API KEY?
+
+  - name: trello_api_token
+    prompt: What is your TRELLO API TOKEN?
+
+  tasks:
+  - name: Install Git
+    ansible.builtin.yum:
+      name: git
+      state: present
+    become: yes
+
+  - name: Install Python3
+    ansible.builtin.yum:
+      name: python3
+      state: latest
+    become: yes
```

Not necessarily a preference, but worth being aware that we can combine these two:
```suggestion
  - name: Install Git & Python3
    ansible.builtin.yum:
      name: 
        - git
        - python3
      state: present
    become: yes
```